modules/indexing/src/test/java/org/apache/ignite/internal/metric/IndexPagesMetricsPageDisplacementTest.java

```diff
@@ -91,11 +81,17 @@ public class IndexPagesMetricsPageDisplacementTest extends GridCommonAbstractTes
     @Override protected IgniteConfiguration getConfiguration(String igniteInstanceName) throws Exception {
         return super.getConfiguration(igniteInstanceName)
             .setFailureHandler(new StopNodeFailureHandler())
+            .setCacheConfiguration(
+                new CacheConfiguration<>(DEFAULT_CACHE_NAME).setIndexedTypes(Integer.class, Person.class)
+            )
             .setDataStorageConfiguration(
                 new DataStorageConfiguration()
+                    // Default values are not suitable for us.
```

Comment:
```suggestion
                    // Default values are not suitable for us because bigger concurrency level means bigger
                    // segment sizes which means no page replacement.
```

Reply:
Fix it.